packages/back-end/src/queryRunners/DimensionSlicesQueryRunner.ts

```diff
@@ -59,6 +59,10 @@ export class DimensionSlicesQueryRunner extends QueryRunner<
           this.integration.runDimensionSlicesQuery(query, setExternalId),
         process: (rows) => rows,
         queryType: "dimensionSlices",
+        querySource: {
+          sourceType: "DimensionSlices",
+          id: this.model.id,
+        },
```

Comment:
Checking my understanding here - there's no page to link back to for DimensionSlices, right? I figured it'd be worth storing the source anyway but open to changing that